CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # PopupBridge iOS Release Notes
 
+## unreleased (v3)
+* Require Xcode 16.2.0+ and Swift 5.10+
+
 ## unreleased
```

Comment:
LOL, as a side note, we should like release a v2 version with this changelog entry before we release v3.